object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -279,6 +304,253 @@ private static String getExactStringValue(Tree tree, ValueAnnotatedTypeFactory f
     return null;
   }
 
+  @Override
+  public void postAnalyze(ControlFlowGraph cfg) {
+    checkAlwaysCall(cfg);
+    super.postAnalyze(cfg);
+  }
+
+  private void checkAlwaysCall(ControlFlowGraph cfg) {
+
+    Set<Pair<BlockImpl, Set<Tree>>> visited = new HashSet<>();
```

Comment:
Let's keep the type of `visited` and `worklist` the same.  So `visited` should be `Set<Pair<BlockImpl, Set<Pair<LocalVariableNode, Tree>>>>`